plugins/outputs/graphite/graphite.go

```diff
@@ -34,6 +50,16 @@ var sampleConfig = `
   template = "host.tags.measurement.field"
   ## timeout in seconds for the write connection to graphite
   timeout = 2
+
+	## Enable secure tunnel
+	ssl_enabled = false
```

Comment:
I think we don't need this option, enabling a secure connection can be based on the tlsConfig being nil or not.